app/packages/looker/src/elements/video.ts

```diff
@@ -593,6 +595,17 @@ export class VideoElement extends BaseElement<VideoState, HTMLVideoElement> {
     return this.element;
   }
 
+  private attachEvents() {
+    for (const eventType in this.events) {
+      this.element.addEventListener(eventType, this.events[eventType]);
```

Comment:
any reason we can't use the abort controller pattern here?

Reply:
These events are short-lived. The looker instance abort controller doesn't apply here, so removing event listener